NanoProd/plugins/DiLeptonBuilder.cc

```diff
@@ -218,22 +242,48 @@ void DiLeptonBuilder<Lepton1, Lepton2>::produce(edm::StreamID, edm::Event& evt,
   for(size_t l1_idx = 0; l1_idx < leptons1->size(); ++l1_idx) {
     const Lepton1& l1 = leptons1->at(l1_idx);
     if(l1_selection_ && !(*l1_selection_)(l1)) continue;
-    if(src1Veto_ && dR_match(l1, *vetoLeptons1, deltaR_thr_)) continue;
+
+    // Get track associated to l1
+    reco::Track track_l1;
+    if (useStandalone_l1_) {
+      // Accept only pat::Muon objects with an outer track
+      if (!(outerTrackAvailable)(l1)) continue; 
```

Comment:
The parentheses around "outerTrackAvailable" are a bit funny - they don't make a difference since the function call takes precedent over the "!" operator, but I suppose they're meant to be around the whole function call expression.